modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -1292,6 +1292,17 @@ The query log includes the GQL error information under the JSON object `errorInf
 * `position` -- The position (a JSON object containing a field for `column`, `offset`, and `line`) in the query where this error occurred.
 * `cause` -- A JSON object containing the `errorInfo` JSON object of the cause of the current `errorInfo` JSON object.
 
+[NOTE]
+====
+The default GQLSTATUS code `50N42` is returned when an exception does not have a GQL object. 
+Starting from Neo4j 5.25, GQL objects are added to exceptions; therefore, you can expect many `50N42` codes. 
+However, it's important not to rely on this default code, as future Neo4j versions might change it by adding an appropriate GQL object to the exception. 
+Additionally, GQL codes for external procedures are not yet stable.
+====
+Neo4j version 5.25 started to add GQL objects to exceptions. Therefore, many `50N42` codes can be expected.
+Note: You should not rely on this default code, since future Neo4j versions might add an appropriate GQL object to the exception (therefore changing this code into another one).
+Also, GQL codes for external procedures are not stable yet.
```

Comment:
These 3 lines can then be removed right?

Reply:
yes, thank you so much